### PR TITLE
Toast message added

### PR DESCRIPTION
**Pull request by azmatroshan**
I have added a toast message. When user wants delete "Use", an alertDialog will pop up to confirm that user really wants to delete "Use" and then if user clicks "Yes" then "Use" will be deleted and a toast message showed up.

screenshots: -

screenshot#1
screenshot#2

### CODE REVIEW

**Comment by LeoColman** [comment]
Detekt is complaining about some issues related to code formatting:

<details>
<summary>Details</summary>

> Task :app:detekt
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:146:126: Argument should be on a separate line (unless all arguments can fit a single line) [ArgumentListWrapping]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:147:9: Missing newline before ")" [ArgumentListWrapping]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:157:54: Argument should be on a separate line (unless all arguments can fit a single line) [ArgumentListWrapping]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:157:103: Argument should be on a separate line (unless all arguments can fit a single line) [ArgumentListWrapping]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:162:1: Missing newline before ")" [ArgumentListWrapping]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:146:88: Missing newline after "," [Wrapping]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:144:16: Missing newline after "(" [Wrapping]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:149:23: Missing newline before ")" [Wrapping]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:146:98: Missing newline before "}" [Wrapping]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:160:40: Missing newline after "{" [Wrapping]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:168:31: Missing newline before "}" [Wrapping]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:135:1: Exceeded max line length (120) [MaximumLineLength]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:140:23: Missing spacing before "{" [SpacingAroundCurly]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:142:28: Missing spacing after "{" [SpacingAroundCurly]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:142:53: Missing spacing before "}" [SpacingAroundCurly]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:163:1: Missing spacing before "}" [SpacingAroundCurly]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:145:1: Unexpected indentation (12) (should be 10) [Indentation]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:145:23: Unexpected indentation (10) (should be 12) [Indentation]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:147:1: Unexpected indentation (10) (should be 12) [Indentation]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:148:3: Unexpected indentation (12) (should be 10) [Indentation]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:149:1: Unexpected indentation (14) (should be 10) [Indentation]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:150:21: Unexpected indentation (8) (should be 10) [Indentation]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:163:23: Unexpected indentation (10) (should be 8) [Indentation]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:164:15: Unexpected indentation (8) (should be 10) [Indentation]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:165:47: Unexpected indentation (8) (should be 10) [Indentation]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:167:1: Unexpected indentation (4) (should be 8) [Indentation]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:140:5: Missing spacing after "if" [SpacingAroundKeyword]
/home/runner/work/Petals/Petals/app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:135:1: Line detected, which is longer than the defined maximum line length in the code style. [MaxLineLength]

</details>

You can run `./gradlew detekt` locally to display all the errors in your command line

File: app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt [hunk 1]
```diff
@@ -132,8 +132,31 @@ fun UseCard(use: Use = Use(), onEditUse: (Use) -> Unit = { }, onDeleteUse: (Use)
...
+
+  Icon(TablerIcons.Trash, null, Modifier.clickable { showDialog.value = true })
+
+  if(showDialog.value){
+    AlertDialog(
+      onDismissRequest = {showDialog.value = false},
+      confirmButton = {
+        Button(onClick = {
+          onDeleteUse(use)
+          Toast.makeText(context, "Deleted Successfully", Toast.LENGTH_SHORT).show() }, colors = ButtonDefaults.buttonColors(Color.White)) {
```
Comment (LeoColman) [inline app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:146]: This string should be Internationalized

File: app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt [hunk 2]
```diff
@@ -132,8 +132,31 @@ fun UseCard(use: Use = Use(), onEditUse: (Use) -> Unit = { }, onDeleteUse: (Use)
...
+          Text(text = "Yes")
```
Comment (LeoColman) [inline app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:147]: This string should be internationalized. I believe 'yes' is already an internationalized string

File: app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt [hunk 3]
```diff
@@ -132,8 +132,31 @@ fun UseCard(use: Use = Use(), onEditUse: (Use) -> Unit = { }, onDeleteUse: (Use)
...
+        }
+      },
+      title = {
+        Row(verticalAlignment = CenterVertically) {
+          Text(text = "Delete")
+          Icon(imageVector = Icons.Outlined.Delete, contentDescription = "delete")
```
Comment (LeoColman) [inline app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:153]: This string should be internationalized, and the same one should be used in both `text` and `contentDescription`

File: app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt [hunk 4]
```diff
@@ -132,8 +132,31 @@ fun UseCard(use: Use = Use(), onEditUse: (Use) -> Unit = { }, onDeleteUse: (Use)
...
+        }
+      },
+      text = { Text(text = "Do you really want to delete this?")},
```
Comment (LeoColman) [inline app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:156]: This string should be internationalized.
Also, 'this' is a bit too generic. We should add the use information instead

File: app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt [hunk 5]
```diff
@@ -132,8 +132,31 @@ fun UseCard(use: Use = Use(), onEditUse: (Use) -> Unit = { }, onDeleteUse: (Use)
...
+      dismissButton = { Button(onClick = { showDialog.value = false }, colors = ButtonDefaults.buttonColors(Color.White)) { Text(text = "No") } }
```
Comment (LeoColman) [inline app/src/main/kotlin/br/com/colman/petals/use/UseCard.kt:157]: This string should be internationalized. I think 'no' is already internationalized too

**Review by LeoColman — CHANGES_REQUESTED** [review]

**Comment by azmatroshan** [comment]
@LeoColman 
I have made changes that you requested and committed it with commit message "added string resources and modified delete feature".

here are the final screenshots  of app: 

Screenshot #1
Screenshot #2

**Review by LeoColman — APPROVED** [review]

**Pull request merged**